modules/objdetect/include/opencv2/objdetect/aruco_detector.hpp

```diff
@@ -0,0 +1,392 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html
+#ifndef OPENCV_ARUCO_DETECTOR_HPP
+#define OPENCV_ARUCO_DETECTOR_HPP
+#include <opencv2/objdetect/aruco_board.hpp>
+#include <opencv2/objdetect/aruco_dictionary.hpp>
+
+/** @defgroup aruco ArUco Marker Detection
+ * This module is dedicated to square fiducial markers (also known as Augmented Reality Markers)
+ * These markers are useful for easy, fast and robust camera pose estimation.
```

Comment:
Different lines, but the same paragraph:
http://pullrequest.opencv.org/buildbot/export/pr/22368/docs/d9/d6a/group__aruco.html#details

Reply:
docs were refactored